nemo/core/classes/common.py

```diff
@@ -657,7 +658,7 @@ class Model(Typing, Serialization, FileIO):
 
     @classmethod
     @abstractmethod
-    def list_available_models(cls) -> Optional[PretrainedModelInfo]:
+    def list_available_models(cls) -> Optional[List[PretrainedModelInfo]]:
```

Comment:
Revert

Reply:
If I correctly understood our conversation in Slack, it's ok.

There were no quotes.

`Optional[List[PretrainedModelInfo]]` to indicate the desired type `List[PretrainedModelInfo]` and `Optional` to maintain the compatibility